### DISCUSSION TITLE
ERROR(541): VENT for ductnode has a DEVC_ID or CTRL_ID, VENT ID:Inlet01

### DISCUSSION THREAD

**Discussion by Prometeo-ianic** [firemodels/fds#13381] [General]
Hi all,
I am running an old model of 6 years ago with the new fds, and I got this error
"ERROR(541): VENT for ductnode has a DEVC_ID or CTRL_ID, VENT ID:Inlet01 ", which I didn't have in my old case study.
Is it now impossible to apply controls to the vent of a ductnode? How can I set the start of a jetfan without a control?
Thanks

**Comment by mcgratta** [comment]
This error was added because people were making VENTs and OBSTs disappear when they were connected to HVAC networks. You control the fan using parameters on the HVAC line.

![image](https://github.com/user-attachments/assets/40b10bac-1db8-4a5d-84a4-644a623df847)

**Reply by Prometeo-ianic** [reply]
How can I define the start of a fan? If I want that the fan initiate to work at 280 s, How can I set it without a logic?
Thank you in advance

**Reply by drjfloyd** [reply]
Please read the User's Guide chapter for HVAC. It discusses this.
